src/d3a/models/strategy/area_agents/two_sided_revenue_fee.py

```diff
@@ -0,0 +1,96 @@
+
```

Comment:
Wouldn't it be better to have all these methods of this module part of a class called something like GridFeeCalculator and then make them like an staticmethod?